### ISSUE TITLE
Schulze_2006: Processing submission

### ISSUE THREAD

**Issue by CaitlanB** [traitecoevo/austraits.build#292]
**Source**: Schulze et al (2006) Species differences in carbon isotope ratios, specific leaf area and nitrogen concentrations in leaves of Eucalyptus growing in a common garden compared with along an aridity gradient *Physiologia Plantarum* 10.1111/j.1399-3054.2006.00682.x
**Custodian**: Ernst-Detlef Schulze
**Handled by**: ?Austraits team member?

- [x] Contact made with contributor (label as `waiting on response` if waiting for files)
- [x] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [x] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [x] Metadata complete, builds into Austraits
- [x] Report reviewed by AusTraits team
- [x] Report sent to contributor ( label as `waiting on response` )
- [x] Response received from contributor
- [x] Files updated & report finalised
- [x] Final checks ( all tests pass )
- [x] Branch merged into master branch (issue `closed` )

NB: 

- if you need to change the config files, e.g. to add a unit conversion or define a new trait. This should be done on the master branch (via pull or rebase)
